README.asciidoc

```diff
@@ -86,119 +80,20 @@ reverse voltage protection for avoiding reverse voltage damaging the
 power outputs.
 
-Connector pinouts:
-
-• Power input
-
-* -  J12 -> Power input positive potential
-* -  J14 -> Power input negative potential
-
-• Motor drivers’ outputs
-
-* -  J10/J13 -> Motor driver output for the first motor
-* -  J7/J8 -> Motor driver output for the second motor
-* -  J5/6 -> Motor driver output for the third motor
-
-* •  J1 thrower connector 1- GND
-
-2- PWM +
-3- Not used 
-4- Not used
+The wires and the mainboard was hidden under the computer. Refer to the picture in the Mechanics. 
 
-* •  J2, J3, J4 encoders 1- GND
-
-2- +3,3 +
-3- TIM_CH1 4- TIM_CH2
-
-[width="99%",cols="29%,26%,22%,23%",options="header",]
+[cols="a,a", frame=none, grid=none]
 |===
-|*COMPONENTS* | | |
-|Name |Comment |Designator |Quantity
-
-|DRV8243 |Motor Driver |U3, U5, U8 |3
-
-a|
-Amphenol 10118193-
-
-0001LF
-
-|USB connector |J9 |1
-
-a|
-CM1223-02SO TVS diode
-
-array
-
-|USB protection |U4 |1
-
-|STM32G441KBT6 |Microcontroller |U1 |1
-
-|MAX14930DASE+ |Isolator (SOIC16 package) |U2, U7 |2
-
-|Micro-Match 215079-4 |Encoder, PWM connectors |J3, J4, J2, J6 |4
-
-a|
-Samtec FTSH-105-01-L-
-
-DV-K
-
-|Programmer connector |J1 |1
-
-|BUK6Y10-30P |Reverse voltage protection |Q1 |1
-
-|NCP164CSN330T1G a|
-Voltage regulator for
-
-microcontroller
-
-|U9 |1
-
-|NCV8730ASN500T1G a|
-Voltage regulator at the
-
-motor driver’s side
-
-|U6 |1
-
-|10k 0603 |Resistor |R1, R5, R10 |3
-
-|1k 0603 |Resistor |R2 |1
-
-|2.7k 0603 |Resistor |R3, R11, R6 |3
-
-|100k 0603 |Resistor |R12, R13, R14 |3
-
-|0.1 uF 0603 |Capacitor (MC) |C1, C5, C6, C2, C3, C4, C7, C8, C9, C10,
-C13 |11
-
-|1 uF 0603 |Capacitor |C11, C12, C14, C15 |4
-
-|10 uF 0805 |Capacitor |C18, C19, C20, C21, C22, C23, C24, C25, C26,
-C27, C28, C29, C30, C31, C32 |*15*
+| image::board_layout.png[]
+| image::board_layout_b.png[]
 |===
 
-Firmware:
-https://github.com/ut-robotics/picr22-team-snax/tree/electronics/firmware
-
-Gerber X2 files:
-https://github.com/ut-robotics/picr22-team-snax/blob/electronics/picr22-team-snax-mainboard_v3011.zip
-
-PCB:
-https://github.com/ut-robotics/picr22-team-snax/tree/electronics/picr22-team-snax-mainboard
-
-== Comments
-
-* Should have chosen different motor driver component with a easier
-footprint to solder. Right now the debugging was time consuming and hard
-because every time I had to resolder the driver.
-* Should have used the possibility to do a bit bigger board than 40x40
-mm. Would have been easier on the mainboard side to solder.
-* Cable management was difficult with Test robot – should have marked
-the cables to have better overview. But it turned out ok.
 
-* {blank}
-+
+=== A short analysis of what was good and bad in your electronics and how to improve.
 
+* The PCB should had a bit more room on the mainboard side for the better soldering 
+* 5V buck converter was very fragile and when even a slight current fluctuation happened, it burned down and also burned down the isolators
```

Comment:
> 5V buck converter

5V converter is linear, not switching regulator.

Reply:
solved